src/browser/screens/Experience.react.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import Component from 'react-pure-render/component';
+
+import ScreenImage from '../components/ScreenImage.react';
+import JobDescription from '../components/JobDescription.react';
+import boundScroll from '../lib/boundScroll';
+import ChevronIcon from '../components/ChevronIcon.react';
+import Alert from '../components/Alert.react';
+import data from '../data/equipment.json';
+
+if (process.env.IS_BROWSER) {
+  require('./Experience.styl');
+}
+
+@boundScroll()
+export default class Experience extends Component {
+  static propTypes = {
+    actions: React.PropTypes.object.isRequired,
+    candidate: React.PropTypes.object.isRequired
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      exp: '',
+      select: false
+    }
+  }
+
+  proceed() {
+    if (!this.state.exp) {
+      this.setState({select: true});
+      setTimeout(() => { this.setState({select: false}); }, 5000);
+    } else {
+      const {actions: {saveExp, nextScreen}} = this.props;
+      saveExp(this.state.exp);
+      nextScreen();
+    }
+  }
+
+  selectLevel(exp) {
+    this.setState({exp: exp}, () => this.proceed());
+  }
+
+  render() {
+    const backpackUrl = {
+      img: "../../../assets/img/mountain-w-flag.svg"
+    };
+    return (
+      <section className="experience-screen screen">
+        <div className="container">
+          <ScreenImage data={backpackUrl} />
+          <div className="right">
+            {this.state.select === true && <Alert desc={"level"}/> }
```

Comment:
Teraz to zauwazylem. Alert powinien byc tylko w jednym komponencie na gorze. Informacje, zeby go wyswietlic i komunikat powinny byc przekazywane do niego (albo ustawic w redux i po zmianie renderowac).